5.py

```diff
@@ -0,0 +1,70 @@
+#1
+n = int(input())
+list = list(map(int, input().split()))
+print(min(list), max(list))
+
+#2
+list = []
+for i in range(9): # 리스트 생성 후 append 함수로 input 추가하기
+    list.append(int(input()))
+print(max(list))
+print(list.index(max(list)) + 1)
+
+#3
+a = int(input())
+b = int(input())
+c = int(input())
+d = list(str(a * b * c))
+for i in range(10):
+    print(d.count(str(i)))
+     # i 이용해서 함 -> count 함수는 str형에만 사용 가능
+
+#4
+list = []
+for _ in range(10):
+    a = int(input())
+    list.append(a % 42)
+list = set(list)
```

Comment:
중복 제거를 위해서 set 을 사용하는 거 아주 좋아요~ 👍 
(참고로 set은 해쉬 구조로 구현되어 있어서 탐색 속도가 굉장히 빠르답니다! 대신 순서는 보장되지 않아요 )

처음부터 set을 선언하고 for문 돌면서 `set.add(a % 42)` 해도 되겠죠??

추가로, Set comprehension..!! 
```suggestion
mySet = {int(input) % 42 for _ in range(10)}
```